[2023-08-13 11:02:39,051] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: submit_pyspark_script_to_emr.upload_script_to_s3 manual__2023-08-13T11:02:35.571002+00:00 [queued]>
[2023-08-13 11:02:39,071] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: submit_pyspark_script_to_emr.upload_script_to_s3 manual__2023-08-13T11:02:35.571002+00:00 [queued]>
[2023-08-13 11:02:39,073] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-08-13 11:02:39,073] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-08-13 11:02:39,074] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-08-13 11:02:39,090] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): upload_script_to_s3> on 2023-08-13 11:02:35.571002+00:00
[2023-08-13 11:02:39,096] {standard_task_runner.py:52} INFO - Started process 178 to run task
[2023-08-13 11:02:39,101] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'submit_pyspark_script_to_emr', 'upload_script_to_s3', 'manual__2023-08-13T11:02:35.571002+00:00', '--job-id', '18', '--raw', '--subdir', 'DAGS_FOLDER/orchestration/submit_spark_job_to_emr.py', '--cfg-path', '/tmp/tmpflukzals', '--error-file', '/tmp/tmpscpwd3ck']
[2023-08-13 11:02:39,103] {standard_task_runner.py:80} INFO - Job 18: Subtask upload_script_to_s3
[2023-08-13 11:02:39,179] {task_command.py:370} INFO - Running <TaskInstance: submit_pyspark_script_to_emr.upload_script_to_s3 manual__2023-08-13T11:02:35.571002+00:00 [running]> on host airflow_scheduler
[2023-08-13 11:02:39,328] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=submit_pyspark_script_to_emr
AIRFLOW_CTX_TASK_ID=upload_script_to_s3
AIRFLOW_CTX_EXECUTION_DATE=2023-08-13T11:02:35.571002+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-08-13T11:02:35.571002+00:00
[2023-08-13 11:02:39,330] {submit_spark_job_to_emr.py:67} INFO - The fetched bucket name is: emr-data-947775527574
[2023-08-13 11:02:39,332] {submit_spark_job_to_emr.py:68} INFO - The fetched local filename is: /opt/airflow/dags/computation/src/pyspark_scripts/read_and_write_back_to_s3.py
[2023-08-13 11:02:39,333] {submit_spark_job_to_emr.py:69} INFO - The fetched S3 file destination is: scripts/read_and_write_back_to_s3.py
[2023-08-13 11:02:39,334] {submit_spark_job_to_emr.py:70} INFO - The fetched spark_conf_map destination is: {'spark.sql.session.timeZone': 'UTC', 'spark.speculation': 'false'}
[2023-08-13 11:02:39,335] {taskinstance.py:1889} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 171, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 189, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/orchestration/submit_spark_job_to_emr.py", line 72, in _local_to_s3
    spark_conf_map = json.loads(spark_conf_map)
  File "/usr/local/lib/python3.7/json/__init__.py", line 348, in loads
    return _default_decoder.decode(s)
  File "/usr/local/lib/python3.7/json/decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/local/lib/python3.7/json/decoder.py", line 353, in raw_decode
    obj, end = self.scan_once(s, idx)
json.decoder.JSONDecodeError: Expecting property name enclosed in double quotes: line 1 column 2 (char 1)
[2023-08-13 11:02:39,351] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=submit_pyspark_script_to_emr, task_id=upload_script_to_s3, execution_date=20230813T110235, start_date=20230813T110239, end_date=20230813T110239
[2023-08-13 11:02:39,368] {standard_task_runner.py:97} ERROR - Failed to execute job 18 for task upload_script_to_s3 (Expecting property name enclosed in double quotes: line 1 column 2 (char 1); 178)
[2023-08-13 11:02:39,393] {local_task_job.py:156} INFO - Task exited with return code 1
[2023-08-13 11:02:39,435] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
